src/librustc_mir/interpret/operand.rs

```diff
@@ -510,7 +504,7 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> EvalContext<'a, 'mir, 'tcx, M>
             // FIXME: do some more logic on `move` to invalidate the old location
             Copy(ref place) |
             Move(ref place) =>
-                self.eval_place_to_op(place, layout)?,
+                self.eval_place_to_op(place)?,
```

Comment:
Hm, but this means we might have to compute the layout even if we already know it, right?

Reply:
To change this we'd need to pass a precomputed layout to `layout_of_local`, which probably doesn't do anything, because any initialized local will already have had called `layout_of_local` on it before. We can still benchmark it and see if it changes anything